wg-code-of-conduct/code_of_conduct.md

```diff
@@ -41,7 +41,9 @@ Examples of unacceptable behavior include:
 PMC members are responsible for clarifying and enforcing our standards of
 acceptable behavior and will take appropriate and fair corrective action in
 response to any behavior that they deem inappropriate, threatening, offensive,
-or harmful.
+or harmful. All PMC members should monitor their spaces for unacceptable
+behavior and notify members. Decisions on actions should be agreed by the PMC
```

Comment:
is "and notify members" a little confusing i.e. which group of members and how? Perhaps "notify the PMC on the private list"?